scripts/wi-setup.sh

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_PATH="$(dirname "${BASH_SOURCE[0]}")"
+
+validate() {
+    # check if user is logged into azure cli
+    if ! az account show > /dev/null 2>&1; then
+        echo -e "\e[31mPlease login to Azure CLI using 'az login'\e[0m"
+        exit 1
+    fi
+}
+
+create_azure_blob_storage_account() {
+    export RESOURCE_GROUP="oidc-issuer"
+    export LOCATION="westus2"
+    if [ $(az group exists --name "${RESOURCE_GROUP}") == 'false' ]; then
+        echo -e "\e[32mCreating resource group '${RESOURCE_GROUP}' in '${LOCATION}'\e[0m"
+        az group create --name "${RESOURCE_GROUP}" --location "${LOCATION}"
+    fi
+
+    export AZURE_STORAGE_ACCOUNT="oidcissuer$(openssl rand -hex 4)"
+    export AZURE_STORAGE_CONTAINER="oidc-test"
```

Comment:
```suggestion
    export AZURE_STORAGE_CONTAINER="oidc"
```

nit: this is for cluster setup, so we can avoid `-test`